src/liboslexec/opclosure.cpp

```diff
@@ -76,7 +76,7 @@ osl_closure_to_string(ShaderGlobals* sg, ClosureColor* c)
     // Special case for printing closures
     std::ostringstream stream;
     stream.imbue(std::locale::classic());  // force C locale
-    print_closure(stream, c, &sg->context->shadingsys());
+    print_closure(stream, c, &sg->context->shadingsys(), false);
```

Comment:
add comment to the `false `parameter identifying the parameter name "treat_string_as_ustring_hash"?